File: php/ram.php
<?php 
 
 class Ram{

    //atribut
    private $capacity;
	private $price;
		
	public function __construct(){
	}
		
	//setter getter 
    public function setCapacity($capacity){
		$this->capacity = $capacity;
	}

    public function getCapacity(){
		return $this->capacity;
	}
		
	public function setPrice($price){
		$this->price = $price;
	}
		
	public function getPrice(){
		return $this->price;
	}
	public function	printRam(){
        echo "Ram~ <br>";
        echo "Capacity  : ".$this->capacity."<br>";
        echo "Price     : Rp ".$this->price. "<br>";
    }
	function __destruct(){
	}
}
?>

		
		

File: c++/Ram.cpp
class Ram{
    private:
        //atribut
        string capacity;
        int price;

    public:
        //konstruktor
        Ram(){
        }
        
        Ram(string capacity, int price){
            this->capacity = capacity;
            this->price = price;
        }

        //setter

        void setCapacity(string capacity){
            this->capacity = capacity;
        }

        void setPrice(int price){
            this->price = price;
        }

        //getter
        string getCapacity(){
            return this->capacity;
        }

        int getPrice(){
            return this->price;
        }

        void printRam(){
            cout << "Ram~ " << endl;
            cout << "Capacity  : "<< this->capacity << endl;
            cout << "Price     : "<< this->price << endl;
        }

        //destruktor
        ~Ram(){
        }

};